docs-user/memory-allocations.md

```diff
@@ -40,17 +42,23 @@ The allocations can be viewed in the call tree, and flame graph, but not the sta
 
 ![A screenshot of the UI to turn on native allocations.](images/allocation-feature.png)
 
-### Limitations in Native Allocation Tracking
+### Native allocation summary strategies
+
+#### Summarize Retained allocations
```

Comment:
Here's some nitpicking on the terms. Not for this patch because obviously this would need some changes in the code too.
Reading the paragraphs, I think we should have slightly different terms.

Especially "Retained allocations" could be simply "Allocations and deallocations". The user doesn't need to understand that we match them, it's implicit.
What do you think?

Reply:
Hmm... I think I prefer the term "retained allocations" as it's more technically correct. I'm not sure which would be clearer to users.